keps/sig-contributor-experience/0005-community-site.md

```diff
@@ -97,14 +114,47 @@ Hopefully the content previously contained in [umbrella issues][] will be tracke
 
 ## Implementation History
 
-## Drawbacks [optional]
+## Drawbacks
+
+The biggest drawback is that this is yet another thing to keep running.
+Currently the markdown files are workable but not super discoverable.
+However, they utilize the familiar mechanisms and do not require extra effort or understanding to publish.
+
+The current mechanisms also scale across orgs and repos.
+This is a strength as the information is close to the code but also a big disadvantage as it ends up being much less discoverable.
+
+## Alternatives
+
+One alternative is to do nothing.
+However, the smattering of markdown through many repos is not scaling and is not discoverable via Google or for other members of the community.
+
+The main alternative here is to build something that is integrated into the user facing kubernetes.io site.
+This is not preferred for a variety of reasons.
+
```

Comment:
I address specific points about the /website flow below without necessarily trying to make an argument in favor of integrating with the current docs site; I think it's helpful to clear up the issues regardless. (Also it's useful for sig-docs to see what misperceptions about docs flow and tooling persist.)